public/locales/en/common.json

```diff
@@ -36,8 +41,14 @@
   },
   "location": {
     "title": "Step 2 of 5: Address",
-    "description": "If you live in NYC, please fill out the home address form.",
+    "internationalInstructions": {
```

Comment:
Render this under `t("location.address.description")` in `src/components/AddressContainer/index.tsx`. We haven't been given guidance on where to render this but it might be a good spot.

Reply:
added